pom.xml

```diff
@@ -66,6 +67,33 @@
 				<artifactId>cf-java-logging-support-logback</artifactId>
 				<version>${cf-java-logging-support.version}</version>
 			</dependency>
+
+			<!-- XSUAA -->
+			<dependency>
```

Comment:
Should we deomnstrate the new XSUAA bom pom here?

Reply:
good idea, I will create a PR..